modules/ensemble/lib/widget/input/dropdown.dart

```diff
@@ -285,6 +285,8 @@ class SelectOneState extends FormFieldWidgetState<SelectOne>
         validatorKey.currentState!.validate();
       }
     });*/
+    textEditingController = TextEditingController(text: widget.getValue());
```

Comment:
also check the getValue code which in turn calls isValueInItems which has the following - 

```dart
      if (_controller.itemTemplate != null) {
        // TODO: we have no way to look into the itemTemplate to see
        // if the value matches one of them. Return true for now
        return true;
      }
```
looks like it just skips over when it is itemTemplate. Not sure how it will impact your fix

Reply:
Umm, Vu wrote this one, and yes we don't have a way by which we can get items from the item template, which is the same issue by which we cannot add autoCorrect with item template